integration-tests/src/test/java/tachyon/client/IsolatedTachyonFileSystemIntegrationTest.java

```diff
@@ -160,14 +160,12 @@ public void lockBlockTest3() throws IOException, TachyonException {
     }
     files.add(TachyonFSTestUtils.createByteFile(mTfs, uniqPath + numOfFiles, fileSize, mWriteBoth));
 
-    for (int k = 0; k <= numOfFiles; k ++) {
-      FileInfo info = mTfs.getInfo(files.get(k));
-      if (k != 0) {
-        Assert.assertTrue(info.getInMemoryPercentage() == 100);
-      } else {
-        CommonUtils.sleepMs(getSleepMs());
-        Assert.assertFalse(info.getInMemoryPercentage() == 100);
-      }
+    CommonUtils.sleepMs(getSleepMs());
```

Comment:
Can add a comment on adding this method? The description in commit message should be good enough.